mkt/developers/views.py

```diff
@@ -553,6 +553,7 @@ def ownership(request, addon_id, addon):
                     thread___addon=addon).delete()
 
         for author in user_form.deleted_objects:
+            author.delete()
```

Comment:
I'd like to verify that but yes. I should name the variable `app_author` or something instead.